perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java

```diff
@@ -0,0 +1,43 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.AttributesManager;
+import cz.metacentrum.perun.core.api.Resource;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.ResourceAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.ResourceAttributesModuleImplApi;
+
+/**
+ * Module for redmine role
+ * Checks if the role has correct value of either Manager, Reporter or Developer
+ * @author Marek Hrasna marekhrasna@outlook.com
+ * @date 04.03.16.
+ */
+public class urn_perun_resource_attribute_def_def_redmineRole extends ResourceAttributesModuleAbstract implements ResourceAttributesModuleImplApi {
+	public void checkAttributeValue(PerunSessionImpl perunSession, Resource resource, Attribute attribute) throws InternalErrorException, WrongAttributeValueException {
+		String role = (String)attribute.getValue();
+
+		if (role == null) {
+			throw new WrongAttributeValueException(attribute, resource, "Attribute value is null.");
```

Comment:
Please change error message to be more comprehensible. 
You can use the same message as it's used few lines later: "Attribute value is invalid.The role can be either Manager, Reporter or Developer"